spec/requests/api/settings_spec.rb

```diff
@@ -38,5 +54,61 @@
 
       expect(response).to have_http_status(:not_found)
     end
+
+    it "supports multiple categories" do
+      stub_api_collection_config("settings", "categories", %w(product authentication server))
+      api_basic_authorize action_identifier(:settings, :read, :resource_actions, :get)
+
+      run_get settings_url
+
+      expect(response.parsed_body).to match(
+        "product"        => normalize_settings(Settings.product),
```

Comment:
I think if we stubbed the `Settings` constant as below, we'd have more control over this. Then you could see in these tests that you're getting back the values you expected. This test gives me low confidence, because it's verifying against something that's not specified (`Settings.product` etc.., whatever these values may be), and it's also munging those values with untested code (`normalize_hash`).